pwd-service/src/main/resources/templates/about.twig

```diff
@@ -5,8 +5,15 @@
     <div class="row mt">
         <div class="col-sm-8 col-sm-offset-2">
             <h1>O projekcie</h1>
-            <p>Lorem ipsum...</p>
-        </div>
+            <p>Celem projektu Polski Walidator Dostępności - PWD On Line jest:</p>
+            <ol>
+                <li>Stworzenie automatycznego mechanizmu walidacji stron internetowych wraz z wygenerowaniem raportu dostęnościci</li>
+                <li>Stworzenie rankingu stron internetowych pozycjonowanych w zależnościci od uzyskanej ilościci punktów raporcie walidacji</li>
+                <li>Systematyczne aktualizowanie rankingu z częstotliwością nie mniejszą niż 7 dni oraz tworzenie i udostęienieienie raportów walidacji</li>
+                <li>Wprowadzenie mechanizmu rywalizacji - zdrowej konkurencji w jak najlepszym udostępnianiu treści internetowych osobom starczym i niepełnosprawnym</li>
+                <li>Udostępnienie szablonów stron internetowych (templatek) do popularnych systemów CMS - Joomla, WordPres i Drupal</li>
```

Comment:
dobra praktyka nakazuje linkować technologie o których piszesz. Wystarczy przy pierwszym wystąpieniu. Wtedy czytelnik jak nie wie co to Joomla to nie musi myszkować tego słowa do google, więc:
`<a href="http://www.joomla.org">Joomla</a>`